languages/kotlin/exercises/concept/basics/.meta/src/main/kotlin/Example.kt

```diff
@@ -0,0 +1,15 @@
+fun remainingMinutesInOven(actualMinutesInOven: Int = 0): Int {
+    val cookingTime = 40
+
+    return cookingTime - actualMinutesInOven
+}
+
+fun preparationTimeInMinutes(numberOfIngredientTypes: Int): Int =
+    5 + numberOfIngredientTypes * 2
```

Comment:
This function uses a different syntax without curly braces. This syntax is not explained in the `introduction.md`, which means that the student probably won't write it like this. In general, the example implementation should only use language features introduced by the exercise or its prerequisites (and their prerequisites, and so on) (see [the docs](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#example-implementation-file)).

This means there are two options here:

1. Use the curly brace syntax here
1. Introduce the non-curly brace syntax

My personal preference would be with the first option, as otherwise the introduction would become just a bit bigger and we're aiming for the introduction to be as concise as possible.

Reply:
Totally agree. Using option 1.